keras_hub/src/models/t5gemma/t5gemma_presets.py

```diff
@@ -10,6 +90,285 @@
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "kaggle://keras/t5-gemma/keras/t5gemma_9b_2b_ul2/1",
+    },
+    "t5gemma_9b_2b_prefixlm": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/2B model with a 9-billion-parameter encoder and "
+                "2-billion-parameter decoder, adapted as a prefix language "
+                "model."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
+    },
+    "t5gemma_9b_2b_ul2_it": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/2B model with a 9-billion-parameter encoder and "
+                "2-billion-parameter decoder, adapted as a UL2 model and "
+                "fine-tuned for instruction following."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
+    },
+    "t5gemma_9b_2b_prefixlm_it": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/2B model with a 9-billion-parameter encoder and "
+                "2-billion-parameter decoder, adapted as a prefix language "
+                "model and fine-tuned for instruction following."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
+    },
+    "t5gemma_9b_9b_ul2": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/9B model with a 9-billion-parameter encoder and "
+                "9-billion-parameter decoder, adapted as a UL2 model."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
+    },
+    "t5gemma_9b_9b_prefixlm": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/9B model with a 9-billion-parameter encoder and "
+                "9-billion-parameter decoder, adapted as a prefix language "
+                "model."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
+    },
+    "t5gemma_9b_9b_ul2_it": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/9B model with a 9-billion-parameter encoder and "
+                "9-billion-parameter decoder, adapted as a UL2 model and "
+                "fine-tuned for instruction following."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
+    },
+    "t5gemma_9b_9b_prefixlm_it": {
+        "metadata": {
+            "description": (
+                "T5Gemma 9B/9B model with a 9-billion-parameter encoder and "
+                "9-billion-parameter decoder, adapted as a prefix language "
+                "model and fine-tuned for instruction following."
+            ),
+            "params": 0,
+            "path": "t5gemma",
+        },
+        "kaggle_handle": "",
     },
 }
```

Comment:
All the 9b checkpoints have been uploaded, you can now add the URLs and the parameter count.